Mybet command

xTomsko opened the pull request:
closes #15

xTomsko committed:
implementation of showbet

xTomsko committed:
clean up

xTomsko committed:
updated readme for showbet

xTomsko committed:
added names to example

xTomsko committed:
refactor

alexsc23 commented:
Add one of these options: https://deno.land/manual/tools/formatter#ignoring-code for successful checks.

xTomsko commented:
> Add one of these options: https://deno.land/manual/tools/formatter#ignoring-code for successful checks.

@alexsc23 Well it is nothing wrong with the changed files. The file that causes the crash is not manually created, it is the bundled JavaScript. For reference #20

Commit:
Added deno config for fmt

Commit:
Use config with git action

Commit:
deno fmt

Commit:
Added lint options to config

alexsc23 commented:
Fixed problem with deno fmt & lint.
But now fails at tests while connecting to mongodb.

xTomsko committed:
deleted useless test

xTomsko committed:
Merge remote-tracking branch 'origin/mybet-command' into mybet-command

xTomsko committed:
fixed typos

xTomsko committed:
disable deno run test

At this point we do not have any functions that require testing. Also all of the current functions require external infrastructure thus making unit tests difficult

Lesoge commented on web/backend/db/mongo.ts:
```diff
@@ -46,7 +46,7 @@ interface UserSchema {
 }
 
-interface BetsSchema {
+export interface BetsSchema {
```
prob extra file for mongo schemas?

xTomsko replied:
sounds like a plan

xTomsko committed:
added schema file for structure

xTomsko committed:
Merge remote-tracking branch 'origin/main' into mybet-command
